source/Calamari.Shared/Integration/Packages/Download/DockerImagePackageDownloader.cs

```diff
@@ -60,6 +63,14 @@ public PackagePhysicalFileMetadata DownloadPackage(string packageId,
                                                            int maxDownloadAttempts,
                                                            TimeSpan downloadAttemptBackoff)
         {
+            if (variables.Get(AuthenticationVariables.FeedType) == FeedType.AwsElasticContainerRegistry.ToString())
+            {
+                var loginDetails = ecrFeedLoginDetailsProvider.GetFeedLoginDetails(variables, username, password).GetAwaiter().GetResult();
+                username = loginDetails.Username;
+                password = loginDetails.Password;
+                feedUri = new Uri(loginDetails.FeedUri);
```

Comment:
Since the call to `GetFeedLoginDetails` is returning a named tuple, you can do all the assignments in one line.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/deconstruct
Once again, since we have a green build, and this comment is purely cosmetic, don't change anything